WinUIGallery/Navigation/NavigationRootPage.xaml

```diff
@@ -67,6 +67,7 @@
                     VerticalAlignment="Center"
                     Style="{StaticResource CaptionTextBlockStyle}"
                     Text="{x:Bind AppTitleText}" />
+                <TextBox x:Name="AppTitleBarTextBox" Width="300" Height="40" Margin="16,0,0,0" Visibility="Collapsed" /> 
```

Comment:
>AppTitleBarTextBox [](http://example.com/codeflow?start=33&length=18)

Does this TextBox do anything? If not, will it be confusing to have a non-functional textbox even if it is only enabled by a titlebar sample?